### PR TITLE
[MAINTENANCE] Update Fabric MeasureAsset.groupby_columns signature

### PR DESCRIPTION
**Pull request by Kilo59**
The `MeasureAsset.groupby_columns` field should take an optional list of `str` rather than a list of tuples.

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: tests/datasource/fluent/great_expectations.yml [hunk 1]
```diff
@@ -237,8 +237,8 @@ fluent_datasources:
           - foo
           - bar
         groupby_columns:
-          - [foo, bar]
-          - [fizz, buzz]
+          - foo
```
Comment (eisber) [inline tests/datasource/fluent/great_expectations.yml]: can you change the test case to 

- foo[Bar]
- 'fizz with space'[Buzz]

File: tests/datasource/fluent/test_fabric.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ class TestFabricPowerBI:
                 "powerbi_measure",
                 {
                     "measure": "my_measure",
-                    "groupby_columns": [("foo", "bar"), ("fizz", "buzz")],
+                    "groupby_columns": ["foo", "bar"],
```
Comment (eisber) [inline tests/datasource/fluent/test_fabric.py]: can you change the test case to 

"groupby_columns": ["foo[Bar]", "'fizz with space'[Buzz]"]

**Review by cdkini — APPROVED** [review]

File: tests/datasource/fluent/test_fabric.py (hunk 2 above)
Comment (Kilo59) [reply tests/datasource/fluent/test_fabric.py]: Done

File: tests/datasource/fluent/great_expectations.yml (hunk 1 above)
Comment (Kilo59) [reply tests/datasource/fluent/great_expectations.yml]: I did it as 
```yaml
- foo[Bar] 
- 'fizz with space[Buzz]'
```

**Pull request merged**